doc/manual/manual.tex

```diff
@@ -5365,6 +5365,91 @@ \subsubsection{Tracking finite strain}
 they can easily be used for computing material model outputs such as the viscosity.  
 
 
+\subsubsection{Reading in compositional initial composition files generated with geomIO}
+\label{sec:geomio}
+\textit{This section was contributed by Juliane Dannberg}
+
+Many geophysical setups require initial conditions with several different materials and complex geometries. 
+Hence, sometimes it would be easier to generate the initial geometries of the materials as a drawing instead 
+of by writing code. The MATLAB-based library geomIO (\url{http://geomio.bitbucket.org/}) provides a convenient tool
+to convert a drawing generated with the vector graphics editor Inkscape (\url{https://inkscape.org/en/}) to a data 
+file that can be read into \aspect{}. Here, we will demonstrate how this can be done for a 2d setup for a model
+with one compositional field, but geomIO also has the capability to create 3D volumes based on series of 2D vector 
+drawings using any number of different materials; and the initial conditions could also be used with particles 
```

Comment:
...different materials; and the... ->  ...different materials. Similarly, initial conditions defined in this way can also...